### PR TITLE
add font library

### PR DESCRIPTION
**Pull request by Feeeenng**
images not font library, may have font error. example
```
java.lang.NullPointerException at sun.awt.FontConfiguration.getVersion
```

Fix #https://github.com/kubesphere/ks-devops/issues/337

### CODE REVIEW

File: tomcat85-java8/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,11 @@ ENV TOMCAT_VERSION 8.5.32
 
 USER root
 
+#Install package
+RUN apk add --update ttf-dejavu fontconfig \
```
Comment (JohnNiang) [inline tomcat85-java8/Dockerfile:28]: Same as above.

File: java11/Dockerfile [hunk 2]
```diff
@@ -18,6 +18,9 @@ LABEL io.k8s.description="Platform for running plain Java applications (fat-jar
       com.yunify.dev-mode="JAVA_DEBUG:false" \
       com.yunify.dev-mode.port="JAVA_DEBUG_PORT:5005"
 
+#Install package
+RUN apt update && apt -y install fontconfig
```
Comment (JohnNiang) [inline java11/Dockerfile:22]: I suggest removing apt cache after installing. Please refer to https://gist.github.com/marvell/7c812736565928e602c4.

File: tomcat85-java11/Dockerfile [hunk 3]
```diff
@@ -24,6 +24,8 @@ ENV TOMCAT_VERSION 8.5.32
 
 USER root
 
+#Install package
+RUN apt update && apt -y install fontconfig
```
Comment (JohnNiang) [inline tomcat85-java11/Dockerfile:28]: Same as above.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: java8/Dockerfile [hunk 4]
```diff
@@ -18,6 +18,10 @@ LABEL io.k8s.description="Platform for running plain Java applications (fat-jar
       com.yunify.dev-mode="JAVA_DEBUG:false" \
       com.yunify.dev-mode.port="JAVA_DEBUG_PORT:5005"
 
+#Install package
+RUN apk add --update ttf-dejavu fontconfig \
```
[the remark this answers is not in the document]
Comment (Feeeenng) [reply java8/Dockerfile:22]: ok

**Comment by JohnNiang** [comment]
Thanks for your first contribution!
/lgtm

**Review by LinuxSuRen — COMMENTED** [review]
/lgtm

This is great. Thanks for your time! @Feeeenng

**Pull request merged**